hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -324,7 +324,7 @@ private boolean metadataTableExists(HoodieTableMetaClient dataMetaClient) throws
     }
 
     if (reInitialize) {
-      metrics.ifPresent(m -> m.updateMetrics(HoodieMetadataMetrics.REBOOTSTRAP_STR, 1));
+      metrics.ifPresent(m -> m.incrementMetric(HoodieMetadataMetrics.REBOOTSTRAP_STR, 1));
       LOG.info("Deleting Metadata Table directory so that it can be re-initialized");
```

Comment:
Are we reporting multiple times for bootstrapping?

Reply:
This metric is for rebootstrapping. `updateMetrics` is updating two metrics - number of times rebootstrap is called and amount of time the action took. There is no good way to calculate the time so I simplified the metric to track only the number of times rebootstrap is called.